rpc/core/routes.go

```diff
@@ -12,6 +12,7 @@ var Routes = map[string]*rpc.RPCFunc{
 	"unsubscribe_all": rpc.NewWSRPCFunc(UnsubscribeAll, ""),
 
 	// info API
+	"health": 				rpc.NewRPCFunc(Health, ""),
```

Comment:
use `go fmt` to format your code or install https://www.jetbrains.com/go/

Reply:
yeap sure, it's still `work in progress`